Loïc-A-N check-same-identification/starter.py

```diff
@@ -0,0 +1,2 @@
+def check_id(l1: list, l2: list, target) -> bool:
```

Comment:
remove type annotations here (this is not the correct syntax for type annotations)